modules/local/download_rsids.nf

```diff
@@ -0,0 +1,24 @@
+process DOWNLOAD_RSIDS {
+  
+  output:
+  tuple  path("rsids-v154-hg${rsbuild}.index.gz"), path("rsids-v154-hg${rsbuild}.index.gz.tbi"), emit: rsids_ch
+
+  script:
+  def rsbuild = params.genotypes_build == 'hg19' ? "${19}" : "${38}"
+
+  """
+  #!/bin/bash
+  if [ $rsbuild -eq 19 ]
+  then
+  wget https://resources.pheweb.org/rsids-v154-hg19.tsv.gz > rsids-v154-hg${rsbuild}.tsv.gz
```

Comment:
I think you have to use  `wget https://resources.pheweb.org/rsids-v154-hg19.tsv.gz -O rsids-v154-hg${rsbuild}.tsv.gz` here.